runtime/core/decoder/CMakeLists.txt

```diff
@@ -7,15 +7,18 @@ set(decoder_srcs
   ctc_endpoint.cc
 )
 
```

Comment:
It's better we move `ov_asr_model.h(cc)` to a new directory, please refer
https://github.com/wenet-e2e/wenet/tree/main/runtime/horizonbpu/bpu

Reply:
Do you mean I create a new dir under ./runtime/openvino/<new_dir> ?